src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/Contracts/BackupInfo.cs

```diff
@@ -48,12 +48,12 @@ public class BackupInfo
         /// <summary>
         /// List of {key: backup path, value: device type}
         /// </summary>        
-        public Dictionary<string, int> arChangesList { get; set; }
+        public Dictionary<string, int> backupPathDevice { get; set; }
```

Comment:
>backupPathDevice [](start = 39, length = 16)

public properties should start start with upper-case letter per .Net coding conventions.  Please switch to BackupPathDevice... also, since this is a collection, should it be BackupPathDevices or some other plural noun?